libdokan/mount_test.go

```diff
@@ -381,14 +403,13 @@ func testOneCreateThenRead(t *testing.T, p string) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer f.Close()
+	defer func() { syncAndClose(t, f) }()
```

Comment:
I suppose calling `syncAndClose` in a closure is because `f` can be overridden to `nil` below. Would you mind a comment here in case it's accidentally changed in the future?